src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,74 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class LoginPage extends BasePage {
+//    public LoginPage(WebDriver driver) {
+//        super();
+//    }
+
+//    public static LoginPage loginPage(){
+//        return new LoginPage(driver);
+//    }
+
+    //LOCATORS
+//    @FindBy(css = "[type='password']")
+//    WebElement passwordInput;
+//    @FindBy(xpath = "//input[@type='email']")
+//    WebElement emailInput;
+
+    //@FindBy(css = "button[type='submit']")
+    //WebElement submitLoginBtn;
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.